samples/FBSDKPlacesSample/FBSDKPlacesSample/PlaceListViewController.m

```diff
@@ -145,6 +162,21 @@ - (void)performSearchForTerm:(NSString *)searchTerm
 
 - (void)searchBarSearchButtonClicked:(UISearchBar *)searchBar
 {
+  if ([searchBar.text stringByTrimmingCharactersInSet:[NSCharacterSet whitespaceCharacterSet]].length == 0) {
```

Comment:
Rather than having this, I'd prefer sticking with the default UISearchBar implementation (which disables the search button until there's some text, even if that text is a space)... If a user wants to search for just a space, that's ok, the SDK will just return things near the location.